coop_product_coefficient/models/product_template.py

```diff
@@ -545,11 +547,11 @@ def _compute_has_theoritical_price_different(self):
     @api.depends("coeff9_inter_sp", "standard_price")
     def _compute_has_theoritical_cost_different(self):
         for template in self:
-            if template.coeff9_inter_sp and (
-                template.base_price or template.alternative_base_price_standard
+            if template.coeff9_inter_sp and (template.base_price or \
+                template.alternative_base_price_standard
             ):
-                template.has_theoritical_cost_different = (
-                    template.standard_price != template.coeff9_inter_sp
+                template.has_theoritical_cost_different = (round(
+                    template.standard_price, 2) != template.coeff9_inter_sp
```

Comment:
Why are you hardcoding the rounding to 2 decimals?
If you need to use rounding, please use the currency to round it, as they might have more decimals in their setup